test/ResourcesClassLibraryWithAttribute/ResourcesClassLibraryWithAttribute.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net451;netstandard1.6</TargetFrameworks>
+    <TargetFrameworks>net452;netstandard1.6</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.6</TargetFrameworks>
```

Comment:
nvm, disregard. I missed that you also changes LocalizationWebsite to conditionally compile.